src/hooks/useGoogleApi.ts

```diff
@@ -0,0 +1,32 @@
+import { GoogleBook } from "../components/API/GoogleBook";
+import { useEffect, useState } from "react";
+import { BooksApiResponse, Items } from "../types/respBooks";
+export const useGoogleBooksApi = (parameter: string) => {
+  const [response, setResponse] = useState<BooksApiResponse[]>([]);
+  const [params, setParamas] = useState(parameter);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  useEffect(() => {
+    const getData = async () => {
+      setIsLoading(true);
+      try {
+        const response = await GoogleBook.get<Items>("/volumes", {
```

Comment:
You can use an es6 destructuring assignment
 here

`const {data: {items}} = await GoogleBook.get`
`// use items`